src/server.c

```diff
@@ -4935,6 +4937,7 @@ sds genRedisInfoString(const char *section) {
             "module_fork_in_progress:%d\r\n"
             "module_fork_last_cow_size:%zu\r\n",
             (int)server.loading,
+            (int)server.async_loading,
```

Comment:
as you suggested, unlike the code inside redis, which will always set the two flags, i agree that the `loading` flag in INFO should be off while doing an async loading.
so you'll need this change:
```suggestion
            (int)(server.loading && !server.async_loading),
            (int)server.async_loading,
```

Reply:
you'll also need to modify `processCommand`:
```c
    if (server.loading && !server.async_loading && is_denyloading_command) {
```